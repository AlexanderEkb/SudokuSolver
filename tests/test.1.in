*6*9*4**6
183*5764*
**46****7
*2***67**
6*****21*
7********
5***7***2
***4*9***
**25**3*4
