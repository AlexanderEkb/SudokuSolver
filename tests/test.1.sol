Ok, let's go!
Loading
∙ 6 ∙ 9 ∙ 4 ∙ ∙ 6 <<ERROR!
ERROR - initial position is wrong
