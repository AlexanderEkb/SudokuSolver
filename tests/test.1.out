ERROR - initial position is wrong